### PR TITLE
Added support for Python2

### PR DESCRIPTION
**Pull request by UtkarshVerma**
This fix prevents the automatic installation of django2 when using `python2`, while installs the latest version available when using `python3`

### CODE REVIEW

**Comment by UtkarshVerma** [comment]
Fiexes #72

File: setup.py [hunk 1]
```diff
@@ -17,9 +18,17 @@ def get_install_requires():
...
+        if sys.version_info[0]<3 :
+            requirements.append("django>=1.11,<2")
+        #Python3 supports latest django
+        else :
+            requirements.append("django")
+             
     return requirements
 
 
+
```
Comment (nemesisdesign) [inline setup.py]: don't add this blank line please

Comment (UtkarshVerma) [reply setup.py]: ok

Comment (UtkarshVerma) [reply setup.py]: Did it.

**Pull request closed without merging**